packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -105,6 +117,31 @@ const setupDriver = async () => {
 		desiredCaps.app = `sauce-storage:Gutenberg-${ safeBranchName }.app.zip`; // App should be preloaded to sauce storage, this can also be a URL
 		if ( isLocalEnvironment() ) {
 			desiredCaps = _.clone( iosLocal );
+
+			const iosPlatformVersions = getIOSPlatformVersions();
+			if ( iosPlatformVersions.length === 0 ) {
+				throw new Error(
+					'No iOS simulators available! Please verify that you have iOS simulators installed.'
+				);
+			}
+			// eslint-disable-next-line no-console
+			console.log(
+				'Available iOS platform versions:',
+				iosPlatformVersions.map( ( { name } ) => name )
+			);
+
+			if (
+				! desiredCaps.platformVersion &&
+				iosPlatformVersions.length > 0
```

Comment:
`length` is already checked above, maybe we can remove this one.

Reply:
Good catch, thanks! I'm going to remove it.

Reply:
Change applied in https://github.com/WordPress/gutenberg/pull/29715/commits/3bf4cc9ac0fa713a6227849c4b9a34e038b85794.